tests/unit/metadata/test_table.py

```diff
@@ -77,3 +80,113 @@ def test_make_ids_unique_field_index_out_of_order(self):
 
         assert new_data['item 1'].equals(data['item 1'])
         assert new_data['item 0'].is_unique
+
+    def test_transform_calls__transform_constraints(self):
+        """Test that the `transform` method calls `_transform_constraints` with right parameters"""
```

Comment:
Could we add some more details about what is being tested, like in the test below?